types/react-timeout/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
not sure i see why you need all these exceptions. the file is tiny. 

In general for a new package, we do not want any exceptions. please remove.

Reply:
I copied that tslint file from another type, already fixed.